api/src/main/resources/messages.properties

```diff
@@ -633,6 +633,11 @@ reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
 reporting.parameter.attributeType=Attribute Type
 reporting.parameter.values=List of values
+reporting.parameter.locations=Locations
```

Comment:
Hi, @djazayeri  I just made some changes and I believe it's consistent now. What do you think?